README.md

```diff
@@ -36,4 +36,42 @@ DD's repo) on a regular basis, especially after releases.
 
 # Supported libraries and frameworks
 
-_Coming soon._
+*Bold* libraries are enabled out of the box and are fully supported.  Non-bold
+libraries are in beta and must be enabled by setting the property
+`-Dsfx.integration.NAME.enabled=true`, where `NAME` is the name given in the
+table below.
+
+| Library | Versions Supported | Instrumentation Name(s) | Notes |
+| ---     | ---                | ---                     | ---   |
+| **Akka HTTP** | 10.0.0+ | `akka-http`, `akka-http-server`, `akka-http-client` | |
+| **Apache HTTP Client** | 4.0+ | `httpclient` | Also supports the DropWizard HTTP Client that subclasses the Apache one |
+| **AWS SDK Client** | 1.10.33+ | `aws-sdk` | |
+| **CouchBase Client** | 2.0.0+ | `couchbase` | |
+| **Cassandra (DataStax client)** | 2.3.0+ | `cassandra` | |
+| **DropWizard Views** | * | `dropwizard`, `dropwizard-view` | |
```

Comment:
What does an asterisk in the versions supported column mean?

Reply:
It means _everything_.